app/controllers/employees_controller.rb

```diff
@@ -6,4 +6,9 @@ def index
     def show
         @employee = Employee.find(params[:id])
     end
+
+    def by_restaurant
+        @restaurant = Restaurant.find(params[:id])
+        @employees = Employee.where(restaurant_id: @restaurant.id)
```

Comment:
yes! took me awhile to figure this out but the HW/today's class should shed some more light. I think this can be refactored to better use ActiveRecord associations